src/components/votingListView/votingHeader.css

```diff
@@ -103,14 +103,6 @@
     border-bottom: solid 1px transparent;
     transition: all ease 250ms;
 
-    &:-moz-placeholder,
```

Comment:
I think the fix is to add an inline disable rule, as the input-placeholder are not supported by autoprefixer. 

https://github.com/postcss/autoprefixer/wiki/support-list

Reply:
it was a wrong rule. it didn't work at all.

Reply:
the webkit one is a valid one, not supported by autoprefixer.

https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-input-placeholder